qa/sc_evm_native_interop.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
```

Comment:
Not directly related to the logic of this test and given the fact that we do not support `DelegateCall`, do you think it is necessary to add a test that verifies the correct throwing of an exception in the case of a contract A that executes `DelegateCall` on the contract B or would it not make sense as there is no risk of unexpected state change?

Reply:
No, you're right. We should add a test verifying correct behavior in case someone tries DelegateCall from the EVM to a native contract.

Reply:
done